src/lib/check.rs

```diff
@@ -151,6 +151,10 @@ pub struct CheckRequest {
     #[clap(short = 'r', long, takes_value = false)]
     /// If present, raw JSON output will be printed instead of annotated text.
     pub raw: bool,
+    #[cfg(feature = "cli"))]
+    #[clap(short = 'c', long, takes_value = false)]
+    /// If present, context (i.e., line number and line offset) will be added to response.
```

Comment:
Does it make sense to compute this always and assume this flag always `true`? I'm just saying this to have a smaller api.

Reply:
Well yes it could, but on the other hand I am trying to keep the `Request` and `Response` structs as close to the LanguageTool API as possible.

Definitely, the CLI API should be kept as small as possible. I still need to think about that, but I might create, e.g., a `--more` flag that adds all the latest features and not add a flag for each feature.

Reply:
This is not release-ready, nor stabilized (I might change the API), but you can now use the `--more-context`, or `-m` for short, option flag to retrieve line number and line offset information. Those can be found in the `response.match.more_context` field.